.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-    "python.pythonPath": "venv\\Scripts\\python.exe",
+    "python.pythonPath": "c:\\Users\\erol1\\git\\equinor\\original_component\\env\\Scripts\\python.exe",
```

Comment:
Amazing

Reply:
hahaha, ups. wonder how that was pushed. Should be in gitignore 😅